C/tasks/north/sending_process/sending.cpp

```diff
@@ -47,14 +56,17 @@ static const string sendingDefaultConfig =
 		"\"type\": \"integer\", \"default\": \"500\" }, "
 	"\"sleepInterval\": {"
 		"\"description\": \"A period of time, expressed in seconds, "
-			"to wait between attempts to send readings when there are no "
-			"readings to be sent.\", \"type\": \"integer\", \"default\": \"1\" }, "
+		"to wait between attempts to send readings when there are no "
+		"readings to be sent.\", \"type\": \"integer\", \"default\": \"1\" }, "
 	"\"north\": {"
 		"\"description\": \"The name of the north to use to translate the readings "
-			"into the output format and send them\", \"type\": \"string\", "
-			"\"default\": \"omf\" }, "
+		"into the output format and send them\", \"type\": \"string\", "
+		"\"default\": \"omf\" }, "
```

Comment:
Really the north plugin is not something we should default. Like the south this should come via the configuration. The process is we create a new service (or task) with a plugin name. It loads that plugin and gets the configuration defaults from that plugin, including the name of the plugin to load. This way the sending process does not need to create the plugin or have a default plugin name like this.